src/Imaginocean.jl

```diff
@@ -210,23 +208,25 @@ function heatsphere!(axis::Axis3, field::Field, k_index=1; kwargs...)
     return axis
 end
 
-function heatmap!(ax::Axis, field::Field, k=1; kwargs...)
+"""
+    function Makie.convert_arguments(P::SurfaceLike, field::Field, depth_level::Int64)
+Convert an Oceananigans.jl `Field` at `depth_level` to arguments that can be plotted as a
+`SurfaceLike` type in Makie.jl.
+"""
+function Makie.convert_arguments(P::SurfaceLike, field::Field, depth_level::Int64)
```

Comment:
how did you figure out that `SurfaceLike ` is the right type to put here?

Reply:
I looked at  the [conversions.jl](https://github.com/MakieOrg/Makie.jl/blob/master/src/conversions.jl) script in the Makie.jl source code. I could not quite figure out what was happening from the [documentation](https://docs.makie.org/stable/documentation/recipes/) for `convert_arguments` but once I saw the conversions that they perform on any data that is passed in for Makie.jl to plot, I was able to figure out what `PlotType` to use for various data plotting. Recently I have been using `convert_argument` methods quite a bit in my own work where I have custom types because it can make plotting quicker than extracting the necessary `Array`s each time.

Reply:
That's quite the Sherlock Holmes situations here!
I couldn't find what was going on from the docs and I thought the problem was me...

Reply:
Yeah it took me a while.. I was hoping to try and add some more explanation to the Makie.jl documentation after my attempts (even just link to the conversions.jl file) but never got round to it.

Reply:
Raise an issue if anything! :)

Reply:
Yes that would be better!